### ISSUE TITLE
is qhypstub implemented in the new code base?

### ISSUE THREAD

**Issue by exkc** [msm8916-mainline/lk2nd#414]

**Comment by TravMurav** [comment]
No, qhypstub support is not implemented in the new lk2nd yet. There is a WIP branch (rebase/hl) with old implementation ported but those changes are unsafe for new lk2nd since they will break most platforms except msm8916.

To re-add it properly, qhypstub itself needs to be updated but we didn't have time to fix it yet.
